agent/client/workflow_send_ds_connection_result.go

```diff
@@ -7,14 +7,14 @@ import (
 	"github.com/kubeshop/tracetest/agent/proto"
 )
 
-func (c *Client) SendDataStoreConnectionResult(ctx context.Context, response *proto.DataStoreConnectionTestResponse) error {
+func (c *Client) SendOTLPConnectionResult(ctx context.Context, response *proto.OTLPConnectionTestResponse) error {
```

Comment:
Is the renaming of this function correct? seems that there is a new file about oltp

Reply:
Not correct, nice catch. Unfortunately this endpoint isn't covered by a test, that's why I didn't get this error :(

Reply:
fixed the mess and also added tests to ensure it will not break in the future